### ISSUE TITLE
Datatable-page does not support custom aria-labels

### ISSUE THREAD

**Issue by HannesEskebaek** [swimlane/ngx-datatable#1508]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Aria-labels are hard-coded for the pager buttons. For example: "go to first page", "go to next page", "page 1" etc.

**Expected behavior**
It should be possible to set custom aria-labels on the datatable-pager buttons so they can be adjusted for non-English speaking users.

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**
Better support for non-English users with screen-readers.

* **Table version:** 13.0.1

* **Angular version:** 6.0.7

* **Browser:** all

* **Language:** all

**Comment by cjessing** [comment]
+1 for this one...
